hera_pspec/utils.py

```diff
@@ -12,6 +12,45 @@
 from datetime import datetime
 
 
+
+def clean_inv_mat(nchan, df, filter_centers, filter_widths, filter_factors):
+    """
+    Computes inverse of clean weights for a baseline.
+    This form of weighting is diagonal in delay-space and down-weights tophat regions
+    Parameters
```

Comment:
insert line breaks (PEP8)

Reply:
I now have two blank lines between the functions I've added. Is this the right number?

Reply:
Yes, but I actually meant for the doc string. Look at other docstrings to see how this is done, but you should keep a space between the function explanation and Parameters section. Ex:
"""
Computes inverse ...
This form of ...

Parameters
-----------
"""
and the same for Returns. I know its nitpicky but I'm just trying to keep everything consistent :)  If you just look at all the other doc strings in this file you'll see the pattern

Reply:
I've added the white space.